cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -509,11 +512,17 @@ class DuplicateMountDirValidator(Validator):
     """
     Mount dir validator.
 
-    Verify if there are duplicated mount dirs between shared storage and ephemeral volumes.
+    Verify if there are duplicated mount dirs.
+    1. Shared storage directories can not be the same with each other.
+    2. Shared storage directories can not be the same with ephemeral storage directories.
+    3. Ephemeral storage directories can be the same with each other, because they are local to compute nodes.
     """
 
-    def _validate(self, mount_dir_list):
-        duplicated_mount_dirs = _find_duplicate_params(mount_dir_list)
+    def _validate(self, shared_mount_dir_list, local_mount_dir_list):
+        duplicated_mount_dirs = _find_duplicate_params(shared_mount_dir_list)
+        for shared_mount_dir in shared_mount_dir_list:
```

Comment:
You don't need this extra logic, you can just do:
```python
duplicated_mount_dirs = _find_duplicate_params(shared_mount_dir_list + local_mount_dir_list)
```

Reply:
Done